### ISSUE TITLE
Facebook post preview image generated.

### ISSUE THREAD

**Issue by BrOrlandi** [nihey/breaking-badify#9]
Would be very nice if the image in the facebook post preview, when sharing the page, shows the message generated with breaking badify.
Maybe it's possible, see [this](http://stackoverflow.com/questions/26630275/base-64-ogimage-to-facebook-share) question on stackoverflow.

The page must be converted into an image and this image should be in some way set as og:image meta tag.
Facebook has [this tool](https://developers.facebook.com/tools/debug/) to debug post preview.

This is a huge challenge. :wink:

**Comment by nihey** [comment]
Indeed, it would be a great enhancement, also bearing a great effort in doing so.

**Comment by nihey** [comment]
This seems to be a great problem for us, we only have a static page here, and it seems Facebook will not execute any JavaScript code from our page (http://stackoverflow.com/a/20969222). To do such thing, we must have a `backend` structure that takes the URL, generate the image and serves the static html along with the generated image.

Even if we are able to build this backend structure, we'd have to deal with Facebook's cache (http://stackoverflow.com/a/15834959). A cachebuster solution would be possible, but I think it would cause higher latency.
